seive.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+function seiveOriginal(n) {
+  let primes = new Array(n - 1).fill(true);
+  [primes[0], primes[1]] = [false, false];
```

Comment:
Just small nitpicking: here variable values are not swapped so it's possible to do this:
```primes[0] = false, primes[1] = false;```
I would say it's easier to read in this case but you decide what's better ;)